pkg/Microsoft.ML.AutoML/Microsoft.ML.AutoML.nupkgproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>ML.NET AutoML: Optimizes an ML pipeline for your dataset, by automatically locating the best feature engineering, model, and hyperparameters</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="1.0.0" />
```

Comment:
My opinion here is that we should be using `<ProjectReference` instead of `PackageReference`.

The reasoning is:

1. That's the way all other libraries in this repo reference each other.
2. If AutoML wants to use a new feature (or needs a bug fix) in the core ML.NET, we will not be able to use it, since we are referencing an older core ML.NET version.
3. It can easily create version mis-match problems. For example, I am an app that consumes `ML.AutoML` NuGet package. By transitivity I also reference `Microsoft.ML` `v1.0.0` and `Microsoft.ML.LightGBM` `v1.0.0`. If I also add a PackageReference to `Microsoft.ML` `v1.1.0`, NuGet will [uplift the reference to the `v1.1.0` version](https://docs.microsoft.com/en-us/nuget/consume-packages/dependency-resolution#nearest-wins). However, no one uplifted `Microsoft.ML.LightGBM` to `v1.1.0`, so I will get `LightGBM` `v1.0.0` matched with a core `Microsoft.ML` `v1.1.0`, which may not be compatible because there are InternalsVisibleTo between `Microsoft.ML` and `Microsoft.ML.LightGBM`.

Reply:
Yes agreed, and done.

Reply:
This doesn't appear to be done. Can you make this use `<ProjectReference`?